### PR TITLE
Metrics Services [old]

### PR DESCRIPTION
**Pull request by whispyy**
redefine metrics services to make structure simpler
implements getServicesInfos()

### CODE REVIEW

File: dist/ovh-api-services.js [hunk 1]
```diff
@@ -4441,119 +4441,83 @@ angular
...
+            },
+            getServiceInfos: {
+                url: "/metrics/:serviceName/serviceInfos",
+                method: "GET",
+                cache: cache
+            },
+            getConsumption: {
+                url: "/metrics/:serviceName/consumption",
+                method: "GET",
+                cache: false
```
Comment (Mboulianne) [inline dist/ovh-api-services.js]: cache: false?  can't we just omit cache property instead?

File: dist/ovh-api-services.min.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
 /*! ovh-api-services - 1.1.10 - 2017-08-16 */
```
Comment (Mboulianne) [inline dist/ovh-api-services.min.js]: is this supposed to be commited?

File: src/metrics/metrics.service.lexi.js [hunk 3]
```diff
@@ -1,23 +1,57 @@
...
+            },
+            getServiceInfos: {
+                url: "/metrics/:serviceName/serviceInfos",
+                method: "GET",
+                cache: cache
+            },
+            getConsumption: {
+                url: "/metrics/:serviceName/consumption",
+                method: "GET",
+                cache: false
```
Comment (Mboulianne) [inline src/metrics/metrics.service.lexi.js]: cache: false? can't we just omit cache property instead?

File: src/metrics/metrics.service.lexi.js [hunk 4]
```diff
@@ -1,23 +1,57 @@
...
+        var resource = $resource("/metrics/:serviceName", {
+            serviceName: "@serviceName"
+        }, {
             query: {
                 method: "GET",
                 cache: queryCache,
                 isArray: true
+            },
+            get: {
+                method: "GET",
```
Comment (Mboulianne) [inline src/metrics/metrics.service.lexi.js]: for readability's sake I think simple API method could be declared as inline objects

File: src/metrics/token/token.lexi.service.js [hunk 5]
```diff
@@ -36,12 +36,12 @@ angular
         });
 
         r.resetAllCache = function () {
```
Comment (neolitec) [inline src/metrics/token/token.lexi.service.js]: I know it's not your code but single-letter variable should be avoided.

Comment (whispyy) [inline src/metrics/token/token.lexi.service.js]: Yes, thanks forgot to edit that one !
Done

File: src/metrics/metrics.service.lexi.js (hunk 4 above)
Comment (whispyy) [inline src/metrics/metrics.service.lexi.js]: done

File: src/metrics/metrics.service.lexi.js (hunk 3 above)
Comment (whispyy) [inline src/metrics/metrics.service.lexi.js]: Yep, done

File: dist/ovh-api-services.js (hunk 1 above)
Comment (whispyy) [inline dist/ovh-api-services.js]: done

**Comment by whispyy** [comment]
I close because i rebase on another branch cf pull : https://github.com/ovh-ux/ovh-api-services/pull/17

**Pull request closed without merging**